set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -365,6 +371,59 @@ private void validateControlledValues(UserSet webUserSet) throws RequestBodyVali
 	}
     }
     
+    /**
+     * The value of isDefinedBy is validated (e.g. search URL https://api.europeana.eu/record/search.json?) 
+     * to point to the Search API.
+     * We make a GET request upon creation to see if the request returns a 200 (meaning is valid).
+     * @param webUserSet
+     * @throws ParamValidationException 
+     * @throws RequestBodyValidationException
+     */
+    private void validateSearchUri(UserSet webUserSet) 
+	    throws ParamValidationException, RequestBodyValidationException {
+	
+	int STATUS_OK = 200;
+
+	if (webUserSet.getIsDefinedBy() != null) {
+            String searchUrl = getBaseSearchUrl(getConfiguration().getSearchApiUrl());
+            String queryUrl = getBaseSearchUrl(webUserSet.getIsDefinedBy());
+            if (!queryUrl.equals(searchUrl)) {	    
+    	        throw new ParamValidationException(UserSetI18nConstants.USERSET_VALIDATION_INVALID_SEARCH_URL,
+		    UserSetI18nConstants.USERSET_VALIDATION_INVALID_SEARCH_URL,
+		    new String[] { WebUserSetModelFields.IS_DEFINED_BY, queryUrl});
+            }
+        	
+    	    String apiKey = getConfiguration().getSearchApiKey();
+    	    int getStatus = 0; 
+    	    try {
```

Comment:
please update the validation according to the last comments in the ticket

Reply:
validation updated according to last comments